pom.xml

```diff
@@ -16,6 +16,7 @@
 	<properties>
 		<enforceFailOnUIDependencyInCore>true</enforceFailOnUIDependencyInCore>
  		<tycho.scmUrl>scm:git:https://github.com/jbosstools/jbosstools-base.git</tycho.scmUrl>
+ 		<sonar.test.inclusions>**/*Test.*,**/test/**/*</sonar.test.inclusions>
```

Comment:
```xml
<include>**/AllTests.class</include>
<include>**/*AllTests*.class</include>
<include>**/*AllBotTests*.class</include>
<include>**/*TestSuite*.class</include>
```
^ don't cover the ones i included into `sonar.test.inclusions`. But i'll add  them too